### DISCUSSION TITLE
Is it possible to enable a "wrap to the left" functionnality ? To fast access last item of a menu

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#681] [Q&A]
Say i want to access the Performance page, to do so i first need to pass TG 1, TG 2, TG 3 up to 8 then Effect and Performance is here.

With a "wrap to the left", you should be able to go LEFT on the first item to reverse scroll items.

**Comment by probonopd** [comment]
That would be useful, yes. Not sure how easy it is to implement though.

**Comment by blixabloxa** [comment]
Yes, that would be a useful feature. I notice that the Performance scrolling wraps around in either direction. Perhaps there is something there which can be utilised for the main menus?

**Comment by probonopd** [comment]
@Onosendaifr please try the build for testing linked in
* https://github.com/probonopd/MiniDexed/pull/690

> the main menu (TG/effects/performance) now wraps around.

Please let us know if it works for you. Thanks!

**Reply by Onosendaifr** [reply]
Which file should I replace? First time i need to update my Mini Dexed.

**Reply by probonopd** [reply]
Best replace all the files except for `mindexed.ini`.

**Reply by Onosendaifr** [reply]
Tested and everything is working fine, thanks !
